Core/MigrationService.php

```diff
@@ -155,7 +156,7 @@ public function parseMigrationDefinition(MigrationDefinition $migrationDefinitio
      *
      * @todo add support for skipped migrations, partially executed migrations
      */
-    public function executeMigration(MigrationDefinition $migrationDefinition, $useTransaction=true)
+    public function executeMigration(MigrationDefinition $migrationDefinition, $useTransaction = true, $defaultLanguageCode = null)
```

Comment:
A minor comment to this: in the future we will add support for custom 'root-user' too, and maybe 'custom-user-contenttype'. I think that we could make thus use of an 'options' param, otherwise the method sig will explode.

Note that since this service is in Core, not API, we could reasonably change its sig when doing major version bumps. Still, it is kinda the main entry point into the bundle for anyone wanting to use it outside of the existing commands. Is it worth changing the sig before releasing 2.0?